code/src/entrance.c

```diff
@@ -111,14 +117,14 @@ void Entrance_Init(void) {
     // Then overwrite the indices which are shuffled
     for (size_t i = 0; i < ENTRANCE_OVERRIDES_MAX_COUNT; i++) {
 
+        if (EntranceIsNull(&rEntranceOverrides[i])) {
+            continue;
```

Comment:
Now that it checks all values in the struct for 0, this could maybe be `break` instead?